Update how ASM works with attack qualification (#17306)
DataDog/documentation: content/en/security/application_security/how-appsec-works.md

{{% asm-protect %}}

## Attack attempt qualification

Leveraging distributed tracing information, attacks attempts are qualified as safe, unknown, or harmful. 
* Attack attempts qualified as safe cannot breach your application, for example, when a PHP injection attack targets a service written in Java. 
* An unknown qualification is decided when there is not enough information to make a definitive judgement about the attack’s probability of success.
* A harmful qualification is highlighted when there is evidence that a code level vulnerability has been found by the attacker.

## Threat monitoring coverage

> Datadog ASM includes over 100 attack signatures that help protect against [many different kinds of attacks][15], including, but not limited to, the following categories:

Reviewer: ```suggestion
Datadog ASM includes over 100 attack signatures that help protect against [many different kinds of attacks][14], including, but not limited to, the following categories:
```

For more information, read [Threat Monitoring and Protection][22].

> ## Attack Attempt Qualification

Reviewer: ```suggestion
## Attack attempt qualification
```

Leveraging distributed tracing information, attacks attempts are qualified as safe, unknown, or harmful. 
* Attack attempts qualified as safe cannot breach your application, for example, when a PHP injection attack targets a service written in Java. 

> * An unknown qualification is decided when there is not enough information to come to make a definitive opinion about the attack’s probability of success.

Reviewer: ```suggestion
* An unknown qualification is decided when there is not enough information to make a definitive judgement about the attack’s probability of success.
```

* A harmful qualification is highlighted when there is evidence that a code level vulnerability has been found by the attacker.

## Threat monitoring coverage

> Datadog ASM includes over 100 attack signatures that help protect against [many different kinds of attacks][15], including, but not limited to the following categories:

Reviewer: ```suggestion
Datadog ASM includes over 100 attack signatures that help protect against [many different kinds of attacks][15], including, but not limited to, the following categories:
```
